subsys/bluetooth/host/conn.c

```diff
@@ -183,16 +183,16 @@ static inline const char *state2str(bt_conn_state_t state)
 		return "disconnected";
 	case BT_CONN_DISCONNECT_COMPLETE:
 		return "disconnect-complete";
-	case BT_CONN_CONNECTING_SCAN:
-		return "connecting-scan";
-	case BT_CONN_CONNECTING_DIR_ADV:
-		return "connecting-dir-adv";
-	case BT_CONN_CONNECTING_ADV:
-		return "connecting-adv";
-	case BT_CONN_CONNECTING_AUTO:
-		return "connecting-auto";
-	case BT_CONN_CONNECTING:
-		return "connecting";
+	case BT_CONN_CENTRAL_CONNECTING:
+		return "central-connecting";
+	case BT_CONN_CENTRAL_SCAN_BEFORE_CONNECTING:
+		return "central-scan-before-connecting";
+	case BT_CONN_CENTRAL_CONNECTING_FILTER_LIST:
+		return "central-connecting-filter-list";
+	case BT_CONN_PERIPHERAL_CONNECTING:
+		return "periph-connecting-adv";
+	case BT_CONN_PERIPHERAL_CONNECTING_DIR_ADV:
+		return "periph-connecting-dir-adv";
```

Comment:
Minor: There's something odd about using the entire word `peripheral` in the macro (where we usually try to be shorter), and then using the short version `periph` in the string where it matters less :)